VERSION

```diff
@@ -1 +1 @@
-0.8.46
+0.9.46
```

Comment:
Request (blocking): Since we are in v0 still, this is a patch release IMO, please adjust